control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1153,14 +1153,6 @@ func (r *HostedControlPlaneReconciler) reconcilePKI(ctx context.Context, hcp *hy
 		return fmt.Errorf("failed to reconcile ingress cert secret: %w", err)
 	}
 
-	// OAuth server Cert
```

Comment:
@relyt0925 is there a reason you need to remove this? isn't the entire reconcile of pki skipped for your use case?

Reply:
No reason... also sorry I was based on the wrong branch

Reply:
Let me fix that in case it makes it easier for you in the future

Reply:
@csrwng fixed that and adjusted the base appropriately